compiler/rustc_parse/messages.ftl

```diff
@@ -309,6 +309,13 @@ parse_inclusive_range_no_end = inclusive range with no end
 parse_incorrect_braces_trait_bounds = incorrect braces around trait bounds
     .suggestion = remove the parentheses
 
+parse_incorrect_restriction = incorrect {$noun} restriction
+    .help = some possible {$noun} restrictions are:
+            `{$keyword}(crate)`: {$adjective} only in the current crate
+            `{$keyword}(super)`: {$adjective} only in the current module's parent
+            `{$keyword}(in path::to::module)`: {$adjective} only in the specified path
+    .suggestion = make this {$adjective} only to module `{$path}` with `in`
```

Comment:
Because of `{$adjective}` this is not translatable. Please duplicate the translations instead (make `parse_incorrect_mut_restriction` and `parse_incorrect_impl_restriction`, etc)

Reply:
Changing this such that the dynamic part is handled entirely within Fluent. Fluent is a bit limited in this area, so I had to create a public name :disappointed:

Other languages can use [selectors](https://projectfluent.org/fluent/guide/selectors.html) to change the wording according to the keyword that is passed.